superset-frontend/src/SqlLab/components/ResultSet/index.tsx

```diff
@@ -95,7 +95,7 @@ interface ResultSetProps {
   showSql?: boolean;
   visualize?: boolean;
   user: UserWithPermissionsAndRoles;
-  defaultQueryLimit: number;
+  defaultQueryLimit?: number;
```

Comment:
Should this be required?

Reply:
ResultSet is used in QueryTable, props passed in does not include defualtQueryLimit. If defaultQueryLimit is required, it will have error at QeuryTable's using ResultSet. Thank you

Reply:
yeah, let's play it safe and pass one in and leave the value as required. 1000 is a good number. thank you!

Reply:
Updated. Thank you very much!